pkg/volume/gce_pd/gce_util.go

```diff
@@ -58,6 +66,12 @@ func (util *GCEDiskUtil) AttachAndMountDisk(pd *gcePersistentDisk, globalPDPath
 	devicePath := ""
 	// Wait for the disk device to be created
 	for {
+		sdAfter, err := filepath.Glob("/dev/sd*")
+		if err != nil {
+			glog.Errorf("Error filepath.Glob(\"/dev/sd*\"): %v\r\n", err)
+		}
+		udevadmChangeToNewDrives(sdBefore, sdAfter)
+
```

Comment:
Not new code below, but this causes kubelet to go to sleep for 1 second, perhaps 10 times in a row, as far as I understand.  Is this in a separate go rountine?